baselines/fedvssl/fedvssl/conf/base.yaml

```diff
@@ -0,0 +1,44 @@
+---
+# this is the config that will be loaded as default by main.py
+# Please follow the provided structure (this will ensuring all baseline follow
+# a similar configuration structure and hence be easy to customise)
+
+
+### hyper-parameters for FL pre-training ###
+pre_training: false
+exp_name: fedvssl_results
+data_dir: /home/data1
+partition_dir: /home/data1/ucf101/annotations
```

Comment:
```suggestion
data_dir: fedvssl/data
partition_dir: fedvssl/data/ucf101/annotations
```

Reply:
that should point to the data dir created when running `dataset_preparation.py`, right?

Reply:
further testing seems to suggest these should be instead:
```
data_dir: fedvssl/data
partition_dir: ucf101/annotations/client_distribution
```